aiidalab_widgets_base/utils/__init__.py

```diff
@@ -31,13 +31,18 @@ def predefine_settings(obj, **kwargs):
 
 def get_ase_from_file(fname, format=None):  # pylint: disable=redefined-builtin
     """Get ASE structure object."""
-    if format == "cif":
-        traj = read(fname, format=format, index=":", store_tags=True)
-    else:
-        traj = read(fname, format=format, index=":")
+    try:
+        if format == "cif":
+            traj = read(fname, format=format, index=":", store_tags=True)
+        else:
+            traj = read(fname, format=format, index=":")
+    except ValueError as e:
```

Comment:
Apparently, we also need to catch a `KeyError`, I just got this when I tried to load a molden file.

```python
/opt/conda/lib/python3.7/site-packages/ase/io/formats.py in read(filename, index, format, parallel, **kwargs)
    597     format = format or filetype(filename)
    598 
--> 599     io = ioformats[format]
    600     if isinstance(index, (slice, basestring)):
    601         return list(_iread(filename, index, format, io, parallel=parallel,

KeyError: 'molden'
```